src/client/opamArg.ml

```diff
@@ -915,10 +920,17 @@ let build_options =
        actually performing them. \
        WARNING: This option is dangerous and likely to break your opam \
        environment. You probably want `--dry-run'. You've been warned." in
+  let ignore_constraints_on =
+    mk_opt ~section ["ignore-constraints-on"] "PACKAGES"
+      "Forces opam to ignore version constraints on all dependencies to the \
+       listed packages. This can be used to actually test compatibility, but \
+       expect builds to break when using this. Note that version constraints \
+       on optional dependencies and conflicts are unaffected."
```

Comment:
err, I meant depopts, not depexts
`depopts: "foo" {>= "3"}` no longer means that the package conflicts with `foo<3` (as it kind of did in opam 1.1, with broken semantics). It merely means that the depopt is ignored if the version doesn't match. So ignoring the constraint would make the depext always honored. Which would only lead to more rebuilds...